students/12th/eunsulim/login.html

```diff
@@ -0,0 +1,96 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="login_box">
+                        <div class="login">
+                            <div class="login_wrap">
+                                <h1>Instagram</h1>
+                                <div class="login_form">
+                                    <form action="" method="">
+                                        <div class="form_box">
+                                            <input type="text" placeholder="전화번호, 사용자 이름 또는 이메일" />
+                                            <input type="password" placeholder="비밀번호" />
+                                        </div>
+                                        <div class="btn_box">
+                                            <button type="submit" class="login_btn" disabled>로그인</button>
+                                        </div>
+                                        <div class="line_wrap">
+                                            <div class="line"></div>
+                                            <div class="line_text">또는</div>
+                                            <div class="line"></div>
+                                        </div>
+                                        <div class="login_pw">
+                                            <button type="button" class="sns_btn">
+                                                <span class="sns_img"></span>
+                                                <span class="sns_login">Facebook으로 로그인</span>
+                                            </button>
+                                            <a href="#" class="">비밀번호를 잊으셨나요?</a>
+                                        </div>
+                                    </form>
+                                </div>
+                            </div>
+                            <div class="join_box">
+                                <p>
+                                    <span>계정이 없으신가요?</span>
+                                    <a href="#">가입하기</a>
+                                </p>
+                            </div>
+                            <div class="download_app">
+                                <p>앱을 다운로드하세요.</p>
+                                <div class="download_img">
+                                    <a href=""><img src="img/appstore_download.png" alt="appstore-download"></a>
+                                    <a href=""><img src="img/googleplay_download.png" alt="google-download"></a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </article>
+            </main>
+            <footer class="footer">
+                <div class="footer_box">
+                    <nav>
+                        <ul>
+                            <li><a href="" target="_blank">소개</a></li>
```

Comment:
```suggestion
                            <li>
                              <a href="" target="_blank">소개</a>
                            </li>
```
부모 자식 관계를 명확히 드러내기 위해서 줄바꿈 해주세요